src/app/courses/courses.component.scss

```diff
@@ -0,0 +1,16 @@
+@import "src/assets/styles/variables.scss";
+
+.row{
```

Comment:
@Yuriy-27 please try to don't style bootstrap classes, good practice is to add some additional specific class name and add styles to it (e.g. class="courses-content row" or class="courses-list col-4")